pages/[...slug].vue

```diff
@@ -0,0 +1,138 @@
+<script setup>
+// GQL
+import _get from 'lodash/get'
+import GENERAL_CONTENT_DETAIL from '../gql/queries/GeneralContentDetail.gql'
+
+// Helpers
+// import stripMeapFromURI from "../utils/stripMeapFromURI"
```

Comment:
This is called by `parseParentPageURL` and should be uncommented.

Reply:
done

Reply:
Weirdly enough importing it caused a build error, and after looking at the nuxt 3 docs and the index page - it seems like utils are auto-imported? It seems like its working without the import in the index.vue homepage as well.